### PR TITLE
update to k8s v1.19

### PR DESCRIPTION
**Pull request by Thor-wl**
update k8s.io to v1.19
related issue: https://github.com/volcano-sh/volcano/issues/1155
Signed-off-by: Thor-wl <1187526662@qq.com>

### CODE REVIEW

**Comment by k82cn** [comment]
let's move related api to `volcano.sh/apis` firstly, and generated client-go by them.

File: pkg/scheduler/framework/session.go [hunk 1]
```diff
@@ -254,8 +258,8 @@ func (ssn *Session) Pipeline(task *api.TaskInfo, hostname string) error {
 }
 
 //Allocate the task to the node in the session
-func (ssn *Session) Allocate(task *api.TaskInfo, hostname string) error {
-	if err := ssn.cache.AllocateVolumes(task, hostname); err != nil {
+func (ssn *Session) Allocate(task *api.TaskInfo, hostname string, podVolumes *volumescheduling.PodVolumes) error {
```
Comment (k82cn) [inline pkg/scheduler/framework/session.go:261]: why do you change this interface?

Comment (Thor-wl) [reply pkg/scheduler/framework/session.go:261]: > why do you change this interface?

k8s v1.19.6 has changed the interface `AssumePodVolume`, arguement `podVolumes` is added and stores pv info at one node

Comment (k82cn) [reply pkg/scheduler/framework/session.go:261]: we need a design doc for that; the API change may break backward compatibility.

Comment (Thor-wl) [reply pkg/scheduler/framework/session.go:261]: > we need a design doc for that; the API change may break backward compatibility.

It's invisible for users and has no influence in backward compatibility

Comment (k82cn) [reply pkg/scheduler/framework/session.go:261]: both Action & Plugins are plugable, we need to keep backward compatibility.

**Comment by Thor-wl** [comment]
@k82cn @william-wang @wpeng102 please take a review

**Pull request closed without merging**